bundles/org.openhab.core.automation/src/main/resources/OH-INF/automation/moduletypes/ThingTriggers.json

```diff
@@ -10,7 +10,7 @@
 					"type": "TEXT",
 					"context": "thing",
 					"label": "Thing",
-					"description": "The UID of the thing.",
+					"description": "The UID of the thing. You can use '*' as wildcard to match multiple things.",
```

Comment:
Not really related to your PR. Just wondering if we already have a way to translate these labels an descriptions.

Reply:
Isn't that what `ModuleTypeI18nService` and `ModuleTypeI18nServiceImpl` is for?

Reply:
Yes, that is right.

All we have to do next is to define the related property files for translation in [JSON based](https://github.com/openhab/openhab-core/tree/main/bundles/org.openhab.core.automation/src/main/resources/OH-INF/automation/moduletypes) module types.